ui/pages/Applications.tsx

```diff
@@ -24,8 +24,8 @@ function Applications({ className }: Props) {
 
   return (
     <Page loading={loading} className={className}>
-      <Flex align wide between>
-        <TitleBar>
+      <Flex align between>
+        <TitleBar style={{ width: "25%" }}>
```

Comment:
Having it full width ends up wrapping the button text. I kind of wish everything had a width prop or something like that where I don't have to make a whole class or component to change one style (usually width or height) you know?

Reply:
Something else is going on if the text *inside* the button is wrapping because of another component's width.

>  I kind of wish everything had a width prop

That would be inline styles...

Reply:
Wow did I just invent inline styles? Great minds. What's the best way for me to add one line of style to something? The TitleBar width is initially 100% and the Button isn't set so it takes up the minimum amount of space.

Reply:
Inline style would be fine here, but I disagree with the `25%` value. I would have to play around and figure out the right values. Maybe `<Flex align end />` instead of `<Flex align between />` would give you what you want?

Reply:
I think it's the Link component that is limiting the width of the button - made a class for it.